### PR TITLE
Bug 1114476 - rename privacy panel to privacy controls

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gasolin gasolin (gasolin)](https://github.com/gasolin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4b61c0d69ef9)

File: apps/settings/elements/root.html [hunk 1]
```diff
@@ -179,7 +179,7 @@ <h2 data-l10n-id="privacyAndSecurity">Privacy &amp; Security</h2>
         </li>
         <li class="privacy-panel-item" hidden>
           <a id="menuItem-privacyPanel" class="menu-item" data-icon="developer" href="#privacyPanel">
-            <span data-l10n-id="privacy-panel">Privacy Panel</span>
+            <span data-l10n-id="privacy-panel">Privacy Controls</span>
```
Comment (crh0716) [inline apps/settings/elements/root.html]: Given the latest l10n guideline, let's remove the text from the element with an l10n id.

**Comment by try-server-hook** [comment]
[gasolin gasolin (gasolin)](https://github.com/gasolin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=3d965cc24326)

**Comment by try-server-hook** [comment]
[gasolin gasolin (gasolin)](https://github.com/gasolin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=08ece536bd56)

**Pull request merged**